actk/steps/make_diagnostic_sheet/make_diagnostic_sheet.py

```diff
@@ -403,12 +244,32 @@ def run(
             # Process each row
             for j, this_metadata in enumerate(metadata):
 
+                # Add some helper columns for subsequent analysis
+
+                # "SubplotNumber" + str(this_metadata) + "/MaxCells" is a new column
```

Comment:
I think it is somewhat odd to include a `/` in a column name. Can you change it an underscore or something?

Reply:
Or ohhhhh I see, its a calculated thing. I would say like `SubplotNumberModCells`?